test/test-parse.cpp

```diff
@@ -361,6 +361,32 @@ TEST(test_parse, parse_invalid_let) {
                               error_invalid_binding_in_let_statement, where,
                               offsets_matcher(&code, 4, 12))));
   }
+
+  {
+    spy_visitor v;
+    padded_string code(u8"let true, true, y\nlet x;");
+    parser p(&code, &v);
+    p.parse_and_visit_statement(v);
+    p.parse_and_visit_statement(v);
+    EXPECT_THAT(v.variable_declarations.size(), 2);
```

Comment:
Nit: Use `EXPECT_EQ` instead of `EXPECT_THAT` where possible. It compiles faster (I think).
```suggestion
    EXPECT_EQ(v.variable_declarations.size(), 2);
```